### ISSUE TITLE
test on my train  model ,result is bad

### ISSUE THREAD

**Issue by keetsky** [Rudrabha/Wav2Lip#164]
![wav2lip_re0](https://user-images.githubusercontent.com/21081197/101319184-7ee62e00-389c-11eb-90b1-e8b732260668.png)
 I trained  my own datas,  then test it , but result is not good,how to fix it ?

**Comment by rhulprksh** [comment]
How did you trained your model?
Can you tell more specific about your dataset?

**Comment by keetsky** [comment]
My Dataset is sililar with LRS2 dataset
my train code is ::
`python -u wav2lip_train.py --data_root datas/qihanEnTrans1_vid_wav2lip_preprocessed/qihanEnTrans1_vid --checkpoint_dir checkpoints/wav2lip_v1/ --syncnet_checkpoint_path checkpoints/pretraind/lipsync_expert.pth   --checkpoint_path checkpoints/pretraind/wav2lip.pth `
train results is ::
![5_2](https://user-images.githubusercontent.com/21081197/101562977-60e70d80-3a03-11eb-8608-aa79abb421ab.jpg)

**Comment by prajwalkr** [comment]
Is there lip-sync in your generated result? At both train and test time?

**Comment by keetsky** [comment]
> Is there lip-sync in your generated result? At both train and test time?
Yes, that is::
Evaluating for 10 steps
 L1: 0.01426870274272832, Sync loss: 1.8362067314711483
 L1: 0.007411555670525717, Sync Loss: 0.1841694246167722: : 23it [00:47,  2.05s/it]

**Comment by prajwalkr** [comment]
The model is overfitting to the training data. Also, your Sync Loss is pretty high. The expert discriminator's eval loss should go down to ~0.25 and the Wav2Lip eval sync loss should go down to ~0.2 to get good results.

**Comment by rebotnix** [comment]
Another reason for the low quality is that the model has just 96px on input resolution. Another feature that has to implemented is to replace the rectangle face detection with a segmentation one. So, even when your loss does <0.25, I do not believe that you can increase the quality that much. Hope that helps you and save you GPU power time.

**Comment by keetsky** [comment]
Thanks!
